### ISSUE TITLE
BLHeli_32 Pass-Through Support

### ISSUE THREAD

**Issue by Turf1975** [ArduPilot/ardupilot_wiki#3293]
[BLHeli_32 Pass-Through Support](https://ardupilot.org/copter/docs/common-blheli32-passthru.html) - Describe the issue/suggestion and improve the title. Please keep a link to the original article if relevant.
Dear developer team,
In my case I had to push the safety switch before the pass-through was working. Maybe this could added as a remark in the documentation.
I am running arducopter 4.05 with hakrc blheli32 escs.
Best regards,

Maarten

**Comment by Hwurzburg** [comment]
Randy, please confirm and I will add...thanks

**Comment by Hwurzburg** [comment]
@rmackay9 bump

**Comment by rmackay9** [comment]
@Hwurzburg, I'm afraid that I really don't know because I don't use BLHeli ESCs very often.  I think I've used them only once and that was on a boat.  Probably @tridge or @andyp1per is in a better place to respond.

**Comment by andyp1per** [comment]
Yes, pretty sure this is true. I have a PR in the works that updates the passthru docs for BLHeli_S as well

**Comment by Hwurzburg** [comment]
@andyp1per did you add this? I dont see it in any of the docs...if not, please confirm that we should add a note to the BL_Heli passthrough page

**Comment by davidbitton** [comment]
@andyp1per it's not working on my Matek H743-SLIM running master.

**Comment by andyp1per** [comment]
What ESC? It’s working for me on this board

**Comment by davidbitton** [comment]
T-Motor F55A Pro II F3 32bit 55A 3-6S 30x30 4in1 ESC w/ BEC

**Comment by Hwurzburg** [comment]
@andyp1per should I add a note in the warning section that safety switch must be off, if used, in order to communicate via passthru?

**Comment by andyp1per** [comment]
@Hwurzburg good idea!

**Comment by andyp1per** [comment]
I actually fixed this on the F55 so closing

**Comment by davidbitton** [comment]
> @andyp1per should I add a note in the warning section that safety switch must be off, if used, in order to communicate via passthru?

Does that apply to situations where there is no safety switch; e.g. when `BRD_SAFETYENABLE` is set to `0`?

**Comment by Hwurzburg** [comment]
no

**Comment by Hwurzburg** [comment]
Closing per andyp1per's note above that this was fixed. Verified: common-blheli32-passthru.rst already documents the safety-switch requirement.
